lib/game.js

```diff
@@ -77,7 +78,7 @@ function checkForCollision() {
   fishies.forEach(function(fish) {
     bullets.forEach(function(bullet) {
       if (collision(fish, bullet)) {
-        boat.score += fish.y;
+        boat.score += parseInt(fish.y);
```

Comment:
Due to subtraction of amplitude/150 in moveFishY, it turned the actual fish.y into a long float. parseInt() is the equivalent of to_i